pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+
+import json
+import time
+import requests
+import base64
+from pokemongo_bot import logger
+from pokemongo_bot.cell_workers.utils import distance, i2f, format_dist, format_time
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot.step_walker import StepWalker
+from pokemongo_bot.worker_result import WorkerResult
+from pokemon_catch_worker import PokemonCatchWorker
+
+
+class MoveToMapPokemon(object):
+
+    def __init__(self, bot, config):
+        self.bot = bot
+        self.config = config
+        self.last_map_update = 0
+        self.pokemon_data = bot.pokemon_list
+        self.caught = []
+        self.unit = self.bot.config.distance_unit
+
+    def get_pokemon_from_map(self):
+        try:
+            r = requests.get('{}/raw_data?gyms=false&scanned=false'.format(self.config['address']))
+        except:
+            logger.log('Could not reach PokemonGo-Map Server', color='red')
+            return []
+
+        raw_data = r.json()
+
+        pokemon_list = []
+        now = int(time.time())
+
+        for pokemon in raw_data['pokemons']:
+            pokemon['encounter_id'] = long(base64.b64decode(pokemon['encounter_id']))
+            pokemon['spawn_point_id'] = pokemon['spawnpoint_id']
+            pokemon['disappear_time'] = int(pokemon['disappear_time'] / 1000)
+
+            pokemon['dist'] = distance(
+                self.bot.position[0],
+                self.bot.position[1],
+                pokemon['latitude'],
+                pokemon['longitude'],
+            )
+
+            if pokemon['disappear_time'] < (now + self.config['min_time']):
+                continue
+
+            if pokemon['dist'] > self.config['max_distance'] and self.config['mode'] != 'snipe':
```

Comment:
Instead of `config['mode'] != 'snipe'` I think this should be `!config['snipe']`